lib/api/BackbeatAPI.js

```diff
@@ -617,6 +620,38 @@ class BackbeatAPI {
         return cb(null, {});
     }
 
+    /**
+     * Validate that the POST request body has the necessary content
+     * @param {String} body - the POST request body string
+     * @return {Object} object containing an error and the request body
+     */
+    _parseScheduleResumeBody(body) {
+        const msg = 'The body of your POST request is not well-formed';
```

Comment:
minor: I think this message is superfluous since the error code already tells this.